rtdata/languages/default

```diff
@@ -297,8 +297,8 @@ HISTORY_MSG_42;Exposure - Tone curve 2
 HISTORY_MSG_43;Exposure - Tone curve 2 mode
 HISTORY_MSG_44;Lum. denoising radius
 HISTORY_MSG_45;Lum. denoising edge tolerance
-HISTORY_MSG_46;Color denoising
-HISTORY_MSG_47;Blend ICC highlights with matrix
+HISTORY_MSG_46;Resize - Long Edge
```

Comment:
What is going on here? You can't just change history messages...

Reply:
Perhaps you can in this case since they are marked obsolete later on. Still, I would like to know for sure, just in case we break some backwards compatibility... If @heckflosse has time, he might want to comment.

Reply:
Not familiar with the works of the message mechanism, and the way you assign/reassign message numbers, reusing obsolete messages seemed to me the safest way to avoid breaking something. I think this could be changed fairly easily if you let me know what it should be.